metomi/rose/host_select_client.py

```diff
@@ -22,7 +22,40 @@
 import psutil
 
 
+def usage():
```

Comment:
This one also appears to hang with no args or opts.

Reply:
This is an internal command which waits for input from stdin.

The best we can do is abort if stdin is not received within a hardcoded period.